docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -420,13 +567,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Classification Model                                 | Accuracy Score\n",
+      "SVC using kernel as a callable function              |       1.00\n",
+      "SVC using precomputed kernel matrix                  |       1.00\n",
+      "QSVC                                                 |       1.00\n",
+      "-------------------------------------------------------------------------\n",
+      "Clustering Model                                     | Accuracy Score\n",
+      "Spectral Clustering using precomputed kernel matrix  |       0.73\n"
```

Comment:
I'm not sure such a table is required here. First of all, both methods, classification and clustering, are in the same table. We can't compare scores directly, the problems are different. Next, even if there's a table, then a few conclusions should be added.